content/master/concepts/pods.md

```diff
@@ -0,0 +1,321 @@
+
+When installing a Provider their service account needs permissions to create and 
+manage managed resources. 
+
+When creating composite resources the Crossplane service account needs
+permissions for the newly created API endpoints. 
+
+The RBAC manager container manages these functions. 
+
+The RBAC manager pod supports two policy options:
+* `All` - The default policy, configures and manages all RBAC capabilities.
+* `Basic` - Only manage policies related to Composite resources. 
+
+Use the `Basic` policy option when manually assigning RBAC policies to
+Crossplane providers and provider resources. The RBAC manager still creates
+cluster roles for composite resources. 
+
+Change from the default `All` to the `Basic` policy with the `-m Basic` pod 
+argument.
+
+To control all RBAC roles and policies related to Crossplane resources and
+providers, you can disable the RBAC manager. 
+
+Disable the RBAC manager after installation by deleting the
+`crossplane-rbac-manager` deployment from the `crossplane-system` namespace.
+
+Disable the RBAC manager before installation by editing the Helm `values.yaml`
+file, setting `rbacManager.deploy` to `false`.
+
+{{< hint "note" >}}
+
+Instructions for changing Crossplane pod settings during installation are in the
+[Crossplane Install]({{<ref "../software/install">}}) section. 
+{{< /hint >}}
+
+## Leader election
+
+By default only a single Crossplane pod runs in a cluster. If more than one
+Crossplane pod runs both pods try to manage Crossplane resources. To prevent
+conflicts Crossplane uses a _leader election_ to have a single pod in control at
+a time. Other Crossplane pods standby until the leader fails. 
+
+{{< hint "note" >}}
+It's possible to run more than one Crossplane or RBAC manager pods for 
+redundancy.
+
+Kubernetes restarts any failed Crossplane or RBAC manager pods.
+Redundant pods aren't required in most deployments.
+{{< /hint >}}
+
+Both the Crossplane pod and the RBAC manager pods support leader elections.
+
+Enable leader elections with the `--leader-election` pod argument.
+
+{{< hint "warning" >}}
+<!-- vale write-good.TooWordy = NO -->
+<!-- "multiple" -->
+<!-- vale write-good.Passive = NO -->
+<!-- allow "is unsupported" --> 
+Running multiple Crossplane pods without leader election is unsupported.
+<!-- vale write-good.Passive = YES -->
+<!-- vale write-good.TooWordy = YES -->
+{{< /hint >}}
+
+
+## Change pod settings
+
+Change Crossplane pod settings either before installing Crossplane by editing
+the Helm `values.yml` file or after installation by editing the `Deployment`.
+
+The full list of configuration options are available in the 
+[Crossplane Install]({{<ref "../software/install">}}) section. 
+
+{{< hint "note" >}}
+
+Instructions for changing Crossplane pod settings during installation are in the
+[Crossplane Install]({{<ref "../software/install">}}) section. 
+{{< /hint >}}
+
+### Edit the deployment
```

Comment:
Could we link to upstream Kubernetes documentation for this? This feels like we're teaching folks how to use Kubernetes.

Reply:
The amount of "teaching kubernetes" is light and just provides practical context to applying the settings discussed earlier.